src/Description.js

```diff
@@ -15,17 +15,23 @@ export default class Description extends Component {
   }
 
   render() {
+    let icon = ' + ';
+
+    if(this.state.showDescription) {
+      icon = ' - ';
+    }
 
     return (
       <li class="description-box"onClick={this.toggleDescription}>
-        <h3>Area: {this.props.matchedType.name} </h3>
-        
+        <h3 class="info">Area: Estes Park Valley </h3> 
+        <h4 class="info">Route Name: {this.props.matchedType.name}</h4>
+        <p class=" info toggle">{icon}</p>
```

Comment:
Consider a different HTML element for this functionality because the content is not really text. It is more of button functionality.